src/main/java/com/spring_boot/config/MailSenderConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.spring_boot.config;
+
+import com.home_work.spring_boot.services.PropertiesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import javax.validation.Validation;
+import javax.validation.Validator;
+import java.util.Map;
+import java.util.Properties;
+
+@Configuration
+public class MailSenderConfig {
+
+    private PropertiesService propertiesService;
+
+    @Autowired
+    public MailSenderConfig(PropertiesService propertiesService) {
+        this.propertiesService = propertiesService;
+    }
+
+    @Bean
+    public JavaMailSender getJavaMailSender() {
```

Comment:
you have spring-boot-starter-mail on classpath. this starter preconfigures JavaMailSender bean for you. all you need to do is to specify relevant properties for mailer in `application.properties`